tests/MyGame/Example/monster.js

```diff
@@ -0,0 +1,362 @@
+// automatically generated, do not modify
+
+// namespace: Example
+
+var flatBuffers = require('flatbuffers');
+
+var imports = {};
+imports.vec3 = require('./vec3');
+imports.test = require('./test');
+imports.monster = require('./monster');
+imports.stat = require('./stat');
+
+
+exports.Monster = function(fb, pos) {
+    this._fb = fb;
+    this._pos = pos;
+    this._table = new flatBuffers.Table(fb, pos);
+};
+
+exports.MonsterBuilder = {};
+
+exports.Monster.prototype.init = function(fb, pos) {
+    this._fb = fb;
+    this._pos = pos;
+    this._table = new flatBuffers.Table(fb, pos);
+    return this;
+};
+
+exports.Monster.prototype.constructor = exports.Monster;
+
+exports.getRootAsMonster = function(buf, offset) {
+    return new exports.Monster(buf, buf.getInt32(offset) + offset);
+}
+
+exports.Monster.prototype.getPos = function(obj) {
+    var o = this._table.getOffset(4);
+    if (o != 0) {
+        if (typeof obj === 'undefined') {
```

Comment:
Yes; I considered this but did it this way to conform more to the C# way of doing it. We would have to provide different names for the ones where we want to populate an object, any suggestions?

initPos(obj) ?